spec/spec_helper.cr

```diff
@@ -34,18 +34,6 @@ Spec.before_each do
   Fiber.current.query_cache = LuckyCache::NullStore.new
 end
 
-class SampleBackupDatabase < Avram::Database
-end
-
-SampleBackupDatabase.configure do |settings|
```

Comment:
I moved this to `config/database.cr` where all the others are. It should have been there in the first place anyway :joy: